lametro/__init__.py

```diff
@@ -69,6 +69,8 @@ def get_organizations(self):
 
         org.add_post("Chief Executive Officer", "Chief Executive Officer")
 
+        org.add_source('https://metro.legistar.com/DepartmentDetail.aspx?ID=28529&GUID=44319A1A-B2B7-48CC-B857-ADCE9064573B', note='web')
```

Comment:
A question! Is there any chance this URL will change?

Reply:
I suppose it could (but that's true of web scraping, in general)....we seem to be placing confidence in the stability of these URLs for the "Crenshaw Project Corporation" and "LA SAFE," too. 

Can you think of another way to assign this source url to the Board of Directors? It could happen in the people scrape, when we build out the web_info dict (e.g., around line 97). Do you have an opinion?

Reply:
That's an idea, but I suppose you're right! It seems more appropriate to do it here, when we make the organizations. I don't suppose we want to add a web scraper here, too...

Reply:
Yes, my above suggestion would not work, any way. So, it seems our only option: create an instance of a LegistarPersonScraper in the `__init__` file. To make DRY code, we could also stick the web scrape logic in a custom `legistar.py` function.....thoughts? (It will add a few seconds to our scrape time, however - if we have to scrape the person page twice.)

Reply:
At the same time...it seems unlikely that the link will change, since its params are based on Legistar identifiers (e.g., ID and GUID). Changing those identifiers would result in some difficult data management for Legistar. 

I also feel not-so-enthusiastic about scraping the person page twice, each scrape.

Reply:
Heard. Ok, I'm alright with this.